TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/Intake.java

```diff
@@ -0,0 +1,40 @@
+package org.firstinspires.ftc.teamcode.subsystems;
+
+import com.acmerobotics.dashboard.FtcDashboard;
+import com.acmerobotics.dashboard.config.Config;
+import com.arcrobotics.ftclib.command.SubsystemBase;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
+import com.arcrobotics.ftclib.hardware.motors.MotorGroup;
+import com.qualcomm.robotcore.hardware.DcMotorEx;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+/**
+ * A simple 2-wheel intake.
+ */
+@Config
+public class Intake extends SubsystemBase {
+    private final MotorGroup intake;
+    public static double speed = 0;
+    private final FtcDashboard dashboard = FtcDashboard.getInstance();
+    private final Telemetry dashboardTelemetry = dashboard.getTelemetry();
+
+    public Intake(final HardwareMap hMap) {
+        intake = new MotorGroup( // NOTE: Left is leader. Physically reverse the motor with bullet connector
+                new Motor(hMap,"intakeLeft", Motor.GoBILDA.RPM_1150),
+                new Motor(hMap,"intakeRight", Motor.GoBILDA.RPM_1150)
+        );
+    }
+
+    public void toggle(){
+        if(speed == 0) speed = .75;
+        else speed = 0;
+    }
```

Comment:
This seems like it would be more readable with the condition and each branch on separate lines.

```suggestion
    public void toggle() {
        if (speed == 0) {
            speed = 0.75;
        } else {
            speed = 0;
        }
    }

```

Reply:
That works. Personally I'm a fan of more condensed if statements, but I'll change it!